### ISSUE TITLE
Positioning of tiles without translate3d?

### ISSUE THREAD

**Issue by tehGoti** [Leaflet/Leaflet#3991]
Hello,

this is my first GitHub or Leaflet contribution, so please tell me if I do something wrong here.

What I discovered:
- Leaflet 0.7 positions tile layer's tiles with left & top style attributes
- Leaflet 1.0 positions tile layer's tiles with translate3d()

While I see the performance benefits of using translate3d() on the whole layer, I don't see benefits for using it on single tiles. This forces the browser to use a whole lot of hardware layers which slows down the map a lot on devices with low graphic's memory (like older iPhones).

I'm developing an app that uses Leaflet with currently three tile layers, additionally several overlays, other UI elements and so on. Touch Feedback alone takes over 100ms with the current Leaflet state. When I rewrote my local Leaflet copy to use "left / top" for single tiles instead of translate3d, I noticed a huge performance increase.

Should this maybe be a configurable feature or be used for tiles generally?

(Note: to notice the performance increase, it's also important not to use fadeAnimation: true)

**Comment by IvanSanchez** [comment]
You can disable all translate3d CSS transforms by setting the javascript global `L_DISABLE_3D` to true. So in a sense, it's already a configurable option.

If you already have worked on some code, can we have a look in the form of a pull request? (or a branch?)

It might be a good feature, but I'd rather focus on finishing up 1.0.0 and then think about incorporating this change.

**Comment by tehGoti** [comment]
I didn't want to disable translate3d completely. Of course it brings a big performance increase, especially when animating elements. But I didn't see the need of using it for every single tile because it forces hardware acceleration for every single tile. At least I hope I got that right.

I just created a pull request. I'm not sure if that's the nicest way to do it - or if it would be better if there are completely distinct functions "setTransform" and "setPosition" that either use transform or left/top, so that the classes calling them have to choose the right function for their need.
